cobalt/cobalt_main_delegate.cc

```diff
@@ -37,6 +38,7 @@ CobaltMainDelegate::~CobaltMainDelegate() {}
 absl::optional<int> CobaltMainDelegate::BasicStartupComplete() {
   base::CommandLine* cl = base::CommandLine::ForCurrentProcess();
   cl->AppendSwitch(switches::kEnableAggressiveDOMStorageFlushing);
+  cl->AppendSwitch(switches::kDisableGpuShaderDiskCache);
```

Comment:
Question: There're other flags (for all platforms) in [cobalt_switch_defaults.cc](https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:cobalt/cobalt_switch_defaults.cc;l=89-97) -- should we consolidate there?